essence-of-live-coding/src/LiveCoding/HandlingState.hs

```diff
@@ -31,7 +31,6 @@ data Handling h where
        , handle :: h
```

Comment:
This can now probably be changed to a regular (non-GADT) definition, and be given a `derive Data`?

Reply:
Hah, forget about `derive Data`, the whole point of your MR is that `Handling` doesn't need a `Data` instance anymore :laughing:

Reply:
Right, I can change it to non-GADT definition.